### PR TITLE
Wai_aria lesson: Rewrite the text to be clearer and fix grammar errors

### PR DESCRIPTION
**Pull request by Eveieve**
## Because

* Some sentences could use more active voice
* It could use a link to a Wikipedia page definition of 'braille' for those who never encountered the word 
* The use of the word 'though' could be reduced
* The sentence could be made more concise
* The lesson contains some grammar errors 

## This PR

* Rewrite some setences to be more concise and to the point 
* Adds a hyperlink to the Wikipedia page for the word 'braille'
* Fix grammar errors 

## Issue

Closes #XXXXX

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: advanced_html_css/accessibility/wai_aria.md
Comment (Lofty-Brambles) [inline advanced_html_css/accessibility/wai_aria.md:61]:
```suggestion
The `aria-label` attribute overrides any native label of an element and modifies its name property in the accessibility tree. It is best used when an element doesn't already have a native label. When added, the string value of `aria-label` attribute  becomes the element's accessible name. **However**, [`aria-label` does not have any effect on some HTML elements](https://github.com/w3c/aria/issues/756), such as `<div>` or a `<span>`. 
```

File: advanced_html_css/accessibility/wai_aria.md
Comment (Lofty-Brambles) [inline advanced_html_css/accessibility/wai_aria.md:77]:
```suggestion
**Do not** use `aria-label` to change how a word is phonetically announced. Certain words may not get announced correctly by a screen reader, and you may have a temptation to try and fix this. You may end up fixing how a word is announced by a screen reader, but that "fix" could end up making no sense when announced by other assistive technologies like a [braille](https://en.wikipedia.org/wiki/Braille) reader.
```
Putting more emphasis on this would be better since it is a common mistake.

Comment (Eveieve) [reply advanced_html_css/accessibility/wai_aria.md:77]: Agreed, thanks!

File: advanced_html_css/accessibility/wai_aria.md
Comment (thatblindgeye) [inline advanced_html_css/accessibility/wai_aria.md:61]:
```suggestion
The `aria-label` attribute overrides any native label of an element and modifies its name property in the accessibility tree. It is best used when an element doesn't already have a native label. When added, the string value of the `aria-label` attribute  becomes the element's accessible name. **However**, [`aria-label` does not have any effect on some HTML elements](https://github.com/w3c/aria/issues/756), such as `<div>` or a `<span>`. 
```

For the last sentence, I think this could be a good use-case for our note boxes, but not really a blocker. @Lofty-Brambles I'm not entirely sure about linking to the W3C issue here... Also not really a blocker, but could we instead just mention the same info ("'aria-label' may not have any - or the intended - effect on certain HTML elements, such as a `div` or `span` without a `role` attribute.")?

File: advanced_html_css/accessibility/wai_aria.md
Comment (thatblindgeye) [inline advanced_html_css/accessibility/wai_aria.md:69]: Let's add a link:

```suggestion
Instead of a screen reader announcing, "X, button", which makes little sense to the user, it would announce, "Close menu, button". Another way you could use `aria-label` is on landmark elements (our [Semantic HTML](https://www.theodinproject.com/lessons/node-path-advanced-html-and-css-semantic-html) lesson gets another shoutout... again!):
```

File: advanced_html_css/accessibility/wai_aria.md
Comment (thatblindgeye) [inline advanced_html_css/accessibility/wai_aria.md:162]:
```suggestion
* [An in-depth guide to ARIA roles](https://www.a11yproject.com/posts/2020-08-20-an-indepth-guide-to-aria-roles/#landmark-roles) goes over an ARIA attribute that alters the semantics of elements, which can allow you to create custom inputs, buttons, and more. Just keep in mind rule number one of the ARIA rules!
```

Not sure "that" is really necessary here.

**Review by thatblindgeye — CHANGES_REQUESTED** [review]

File: advanced_html_css/accessibility/wai_aria.md
Comment (Eveieve) [reply advanced_html_css/accessibility/wai_aria.md:162]: That's my mistake.

**Comment by Eveieve** [comment]
Hi @thatblindgeye, I wanted to ask, when you make a code suggestion, should I wait for other maintainers/contributors to review it or can I just `commit suggestion` if I agree with your suggestion?

File: advanced_html_css/accessibility/wai_aria.md
Comment (Lofty-Brambles) [reply advanced_html_css/accessibility/wai_aria.md:61]: Sure thing! I found the conversation interesting, since it came from the very same confusion, which's why I linked it here. Not a blocker if you think it is not important. @thatblindgeye

Comment (thatblindgeye) [reply advanced_html_css/accessibility/wai_aria.md:61]: I think my main worry is whether it will bog users down on a conversation that they may not understand the context of (could just be an invalid worry 😆 ). I don't really feel super strong either way; worse case is we keep the link and if any users start questioning it we can replace it in the future.

**Comment by thatblindgeye** [comment]
@Eveieve Good question! Usually I try to say something like "would like feedback from so and so" to directly ask for more input, though sometimes it can be kind of in the air. I'd say if you agree with it and it doesn't seem readily obvious that more input is needed, go ahead and make suggested changes.

Worst case scenario is you end up needing to undo that change/make another update

File: advanced_html_css/accessibility/wai_aria.md
Comment (Eveieve) [reply advanced_html_css/accessibility/wai_aria.md:61]: I do not feel so strongly about whether to keep the link or not either. I think the learners will know not to spend too much time understanding the reasoning behind this. If they *want* to dive deeper into the issue, they've got their source.

**Comment by Eveieve** [comment]
@thatblindgeye Thanks for answering my question! I'm fairly new to contributing so wanted to make sure I'm doing it in the way it's supposed to be done.

**Review by thatblindgeye — APPROVED** [review]
Looks good! Thanks for the updates on these lessons 💪🏼

**Pull request merged**
